bit32.go

```diff
@@ -63,7 +63,7 @@ func fieldArguments(l *State, fieldIndex int) (uint, uint) {
 var bitLibrary = []RegistryFunction{
 	{"arshift", func(l *State) int {
 		r, i := CheckUnsigned(l, 1), CheckInteger(l, 2)
-		if i < 0 || 0 == (r&(1<<(bitCount-1))) {
+		if i < 0 || (r&(1<<(bitCount-1)) == 0) {
```

Comment:
I found this more readable before 🤷

Reply:
I was simply removing warning in the code. I personally don't mind either way. Do you have more context on the whole yoda case lint warning? What's Shopify's stance on using it? I can revert these and disable the lint warning.